src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/MethodParamPadCheckTest.java

```diff
@@ -200,4 +200,13 @@ public void testInvalidOption() throws Exception {
         }
     }
 
+    @Test
+    public void testMethodParamPadSetOptionTrim() throws Exception {
```

Comment:
Please remove check name from method name. We don't need to duplicate it since the class has the method name already. Input is required to have it to prevent duplicate file names.

Reply:
done